openslides/utils/settings.py.tpl

```diff
@@ -82,10 +82,18 @@ if use_redis:
     # Django Channels
 
     # https://channels.readthedocs.io/en/latest/topics/channel_layers.html#configuration
-
-    CHANNEL_LAYERS['default']['BACKEND'] = 'channels_redis.core.RedisChannelLayer'
-    CHANNEL_LAYERS['default']['CONFIG'] = {"capacity": 100000}
-
+    CHANNEL_LAYERS = {
+        "default": {
+            "BACKEND": "channels_redis.core.RedisChannelLayer",
+            "CONFIG": {
+                "hosts": [("localhost", 6379)],
+                "channel_capacity": {
```

Comment:
What is the reason, that you use different values for the http.request, and the http.response channel?

And can you show me the documentation, that it is even possible to configure the channel_layer like this? As far as i understand, channel 2 does not use the channel_layer for http.request and http.response.

And what about other channel_layers like websocket?

Reply:
I basically took the part directly from the documentation. Over at their git repository: https://github.com/django/channels_redis#usage

Reply:
The issue with the string expection was resolved and I changed it back to the general `capacity` call. Still works fine.